astroquery/ipac/irsa/tests/test_most.py

```diff
@@ -0,0 +1,249 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import io
+import os
+import pytest
+
+from astropy.io import votable
+from astropy.table import Table
+from astroquery.utils.mocks import MockResponse
+from astroquery.ipac.irsa import Most
+from astropy.utils.diff import report_diff_values
```

Comment:
hmm, interesting find, I wasn't aware of this one.

Reply:
Secret wonders hide in the depths of various utils modules in this wide bright world.